### PR TITLE
SEM-157 - Deploy pgbouncer for Postgres Connection Pooling

### PR DESCRIPTION
**Pull request by mike-sellers**
This PR contains updates to the manifests/deployment.yml to incorporate pgbouncer connection pooling.

### CODE REVIEW

File: manifests/deployment.yml [hunk 1]
```diff
@@ -29,6 +29,48 @@ spec:
...
+        ports:
+        - containerPort: 5432
+        resources:
+          requests:
+            cpu: 500m
+          limits:
+            cpu: 2048m   
+            memory: 4096M
+        env:
+            - name: NODE_ENV
```
Comment (alihacks) [inline manifests/deployment.yml:43]: node env should not be needed for pgbouncer

File: manifests/deployment.yml [hunk 2]
```diff
@@ -40,10 +82,7 @@ spec:
             - name: NODE_ENV
               value: stage
             - name: DB_HOST
-              valueFrom:
-                secretKeyRef:
-                  name: semesterly-secrets
-                  key: DB_HOST
+              value: localhost
```
Comment (alihacks) [inline manifests/deployment.yml:85]: I think we want to set this to name of the container = pgbouncer, does that work?

Comment (mike-sellers) [reply manifests/deployment.yml:85]: This did not work.  When attempted, the container was unable to resolve pgbouncer for the database connection.

File: manifests/deployment.yml (hunk 1 above)
Comment (mike-sellers) [reply manifests/deployment.yml]: Addressed in follow-up commit

**Pull request merged**